moped-database/views/project_list_view.sql

```diff
@@ -1,6 +1,5 @@
--- latest version 1706897389736_fix_sub_comp_date_dash
```

Comment:
i like being able to see when it was last updated, not sure if omitting this from the file was intentional

Reply:
Thank you for pointing that out Chia, that's helpful. I removed it, and didn't replace it because I am planning on including that comment hint about what's the most recent migration that sets a view as part of my work today on https://github.com/cityofaustin/atd-data-tech/issues/15704, but I should have updated it instead in this PR. I'll stick it back in there - thank you again.